packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/ConsumptionHistory.tsx

```diff
@@ -29,13 +29,9 @@ export const ConsumptionHistory: React.FC<ConsumptionHistoryProps> = ({
 }) => {
   const t = useTranslation('replenishment');
   const theme = useTheme();
-  const d = useFormatDate();
+  const { customDate } = useFormatDateTime();
   const { data, isLoading } = useRequestLineChartData(id);
-  const dateFormatter = (date: string) =>
-    d(new Date(date), {
-      month: 'short',
-      day: '2-digit',
-    });
+  const dateFormatter = (date: string) => customDate(date, 'dd MMM');
```

Comment:
Does the same thing as what it replaces, just uses `date-fns` functions (localised) rather that Intl.DateTime

Reply:
potentially a candidate for a preset date format too?

Reply:
Yup: 78394429